docs/integrations/llms/anthropic.md

```diff
@@ -103,3 +103,23 @@ Shows up like this in Logfire:
   ![Logfire Anthropic Streaming](../../images/logfire-screenshot-anthropic-stream.png){ width="500" }
   <figcaption>Anthropic streaming response</figcaption>
 </figure>
+
+# Amazon Bedrock
+You can also log Anthropic LLM calls to Amazon Bedrock using the `AmazonBedrock` and `AsyncAmazonBedrock` clients.
+
+To use this integration, you'll need the AWS SDK dependencies:
+- `boto3`
+- `botocore`
```

Comment:
Let's stick to only recommending the `bedrock` extra so that we can add other dependencies to that if needed.

Reply:
actually, do we need an extra at all? boto3 and botocore are required to use anthropic bedrock at all, not just for the logfire part.

Reply:
I don't think we even need to mention `anthropic[bedrock]`, it's not like we tell users to install `anthropic`. We assume that they already know how to use `AnthropicBedrock` and just need to instrument it.